spec/features/answers/attach_file_spec.rb

```diff
@@ -0,0 +1,34 @@
+require_relative '../feature_helper'
+
+feature 'Attach file to answer', %q{
+  In order to get illustrate answer
+  As an answer author
+  I can attach file to answer
+} do
+
+  given(:user) { create(:user) }
+  given(:question) { create(:question) }
+
+  background do
+    sign_in user
+    visit question_path(question)
+  end
+
+  scenario 'user attach file while create answer', js: true do
+
+    fill_in 'Type your answer', with: "You must going to sleep in this situation"
+
+    # файл почему-то не прикрепляется
+    attach_file 'attachment', "#{Rails.root}/spec/spec_helper.rb"
+
+    click_on 'Add an answer'
+    expect(page).to have_content 'Your answer successfully created'
+    within '.answers' do
+      expect(page).to have_content "You must going to sleep in this situation"
+
+      # падает, так как файл отсутствует
+      expect(page).to have_link 'spec_helper.rb', href: '/uploads/attachment/file/1/spec_helper.rb'
```

Comment:
Failure/Error: expect(page).to have_link 'spec_helper.rb', href: '/uploads/attachment/file/1/spec_helper.rb'
       expected to find link "spec_helper.rb" with href "/uploads/attachment/file/1/spec_helper.rb" but there were no matches